server/src/main/resources/db/migration/memory/V5_0__Additional_Data_Storage_Info.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE object_permissions ADD COLUMN granter_address VARCHAR(44) NOT NULL;
```

Comment:
had to split these migrations out as the syntax is different for sqlite... also, didn't see a need to do the granter address insertion, as these are more local testing dbs...